Gems/Atom/Feature/Common/Assets/Shaders/Reflections/ReflectionProbeRenderObjectSrg.azsli

```diff
@@ -13,12 +13,9 @@
 ShaderResourceGroup ObjectSrg : SRG_PerObject
 {
     row_major float3x4 m_modelToWorld;
-
-    float3 m_aabbPos;
-    float3 m_outerAabbMin;
-    float3 m_outerAabbMax;
-    float3 m_innerAabbMin;
-    float3 m_innerAabbMax;
+    row_major float3x4 m_modelToWorldInverse; // does not include extents
```

Comment:
Why not just embed the `ReflectionProbeData` struct here instead of duplicating its members?

Reply:
There are actually two separate Srgs used in different passes - one in the specular reflection pass, and the other in the forward pass.  The struct is really only needed in the forward pass Srg since it's co-located with some other data for the draw call.  There's also a small functional difference between the two, so I'd prefer to keep them separate.